packages/api/src/email-campaign/email-campaigns.service.ts

```diff
@@ -125,6 +125,6 @@ export class EmailCampaignsService {
             } while (currentOffset < totalContacts);
         }
```

Comment:
not inthis PR:

but as discussed, this function is wrong. the sendBrevoCampaign has to be put out of the while loop.